docs/actions/live-events.mdx

```diff
@@ -10,13 +10,52 @@ You can stream events (sign-ups, logins, machine-to-machine tokens issued, and m
 your Ory Network project, to your own infrastructure. Pipe those events into your own data warehouse, data lake, or flavor of
 choice, and use them to power your own analytics, dashboards, data science, and more.
 
-Live event streams are available for Ory Network enterprise contracts. Talk to your account manager or reach out
-[directly](https://www.ory.sh/contact/) to find out more.
+Live event streams are available for Ory Network enterprise contracts. Talk to your account manager or
+[reach out directly](https://www.ory.sh/contact/).
 
-## Event descriptions
+## Event shape
 
-Ory emits events for many different actions. The following is a list of all events that are currently supported. This list is not
-exhaustive, and there might by additional events being emitted.
+Ory emits events for many different actions. All events abide by the same schema:
+
+```json
+{
+  "name": string,       // The unique name of the event (e.g., "IdentityCreated").
+  "version": string,     // The numerical schema version of the event as a string.
+  "timestamp": string,  // ISO 8601 formatted timestamp when the event occurred.
+  "projectId": string,  // The ID of the Ory project the event belongs to.
+  "eventAttributes": map<string, string> // Additional event-specific attributes.
+}
+```
+
+Example:
+
+```json
+{
+  "name": "WebhookSucceeded",
+  "version": "1",
+  "timestamp": "2025-05-16 11:18:56",
+  "eventAttributes": {
+    "GeoLocationCity": "Munich",
+    "WebhookID": "",
+    "WebhookTriggerID": "18093e08-0f72-445f-aacb-151fc777a232"
+  }
+}
+```
+
+**Important:**
+
+- Per the JSON specification, ordering of keys in objects is not guaranteed and should not be relied upon.
+- New keys can appear at any time, consumers should be designed to handle this.
+- The exact content in the `eventAttributes` field is currently not documented and not considered stable. That means that no
+  backwards compability guarantees are made at the moment for them.
+
+If you intend to write a consumer that needs a detailed schema and stability guarantees about the `eventAttributes` field, please
+[reach out](https://www.ory.sh/contact/).
+
+## Event names
+
+The following is a list of all events that are currently supported. The first column corresponds to the `name` field in an event.
+New event names may be added in the future, in which case this list will be updated accordingly.
```

Comment:
```suggestion
```

should be obvious, not sure if we need to state this here, I leave it up to you :)